src/Hosting/Abstractions/src/MetricsPrototype/IMeterFactory.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Extensions.Metrics;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+#pragma warning disable RS0016 // Add public types and members to the declared API
+public sealed class MeterFactoryOptions
+{
+    public string? ContainerId { get; set; }
+    public IDictionary<string, string> DefaultTags { get; } = new Dictionary<string, string>();
+}
+
+public sealed class MeterOptions
```

Comment:
Yes, when meter has a way to configure default tags (or any other configuration on meter) then it would be added to the options.

Default tags here would be combined with the default tags configured in the container.